core/backend/src/IModelDb.ts

```diff
@@ -833,6 +836,17 @@ export abstract class IModelDb extends IModel {
     return undefined;
   }
 
+  public static async findOrOpen(requestContext: AuthorizedClientRequestContext, iModel: IModelRpcProps, syncMode: SyncMode): Promise<IModelDb> {

```

Comment:
I don't understand why `findOrOpen` is necessary. It only applies to `IModelReadRpc`, but I don't think it's right there either. Why is that necessary for reattaching a token?